src/style.css

```diff
@@ -1,3 +1,66 @@
+:root {
+  --main-color: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
+.header {
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  height: 60px;
+}
+
+.nav {
+  display: flex;
+  height: 100%;
+  cursor: pointer;
+}
+
+.nav__ul {
+  list-style-type: none;
```

Comment:
it's better to use `list-style: none`, it will remove all list dedicatetd styles